research/src/pages/popup/popover.research.explainer.mdx

```diff
@@ -170,19 +172,19 @@ The declarative trigger attributes can also be accessed via IDL:
 
 ```javascript
 // These set the IDREF for the target element, and not the element itself:
-myButton.popoverToggleTarget = idref;
```

Comment:
Any reason to remove the `;`? I've always thought JS's optional `;` was a huge mistake.